src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -629,7 +629,7 @@ public void setHaltOnException(boolean haltOnException) {
     }
 
     /**
-     * Set the tab width to report errors with.
+     * Set the tab width to report audit event with.
```

Comment:
If it required to be plural ?  is it sounds more natural in plural form ?

Reply:
To me, this makes it sound like the tab width is being used for ALL audit events. Since more than 1 is involved, it should be plural and sounds more natural this way.
If this tab width was only being used for 1 audit event and, for example, being reset to the default afterwards, the singular version should be worded like `an audit event`.

Reply:
done.